components/block-actions.tsx

```diff
@@ -35,7 +36,7 @@ export function RunCodeButton({
   const codeContent = block.content;
 
   const updateConsoleOutput = useCallback(
```

Comment:
This function isn't being used

Reply:
This has also been fixed in the following commit: https://github.com/vercel/ai-chatbot/commit/df25edf2519d70dfe28238438b3440f28dc21e76